### PR TITLE
Move configuration into ConfigService

### PR DESCRIPTION
**Pull request by bspindler**

### CODE REVIEW

**Comment by TheConnMan** [comment]
Going to get some merge conflicts with #16.

**Comment by bspindler** [comment]
I started this as part of the encryption work which I have to reconsider so cleaned up the branch and submitted.  If you want to merge and fix conflicts with #16 on your side, I'm good.  If you want to decline this PR in favor of what you are doing, I'm also good.  Just let me know.

File: ts/services/ConfigService.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs';
```
Comment (TheConnMan) [inline ts/services/ConfigService.ts]: This file is no longer needed, correct?

**Comment by TheConnMan** [comment]
I'll resolve conflicts in my PR after this is merged.

File: ts/services/ConfigService.ts (hunk 1 above)
Comment (bspindler) [reply ts/services/ConfigService.ts]: right

**Pull request merged**